pkg/database/mysql/index.go

```diff
@@ -0,0 +1,35 @@
+package mysql
+
+import (
+	"fmt"
+	"strings"
+
+	schemasv1alpha2 "github.com/schemahero/schemahero/pkg/apis/schemas/v1alpha2"
+	"github.com/schemahero/schemahero/pkg/database/types"
+)
+
+func RemoveIndexStatement(tableName string, index *types.Index) string {
+	return fmt.Sprintf("alter table %s drop index %s", tableName, index.Name)
+}
+
+func AddIndexStatement(tableName string, schemaIndex *schemasv1alpha2.SQLTableIndex) string {
+	unique := ""
+	if schemaIndex.IsUnique {
+		unique = "unique "
+	}
+
+	name := schemaIndex.Name
+	if name == "" {
+		name = types.GenerateIndexName(tableName, schemaIndex)
```

Comment:
there is no workaround if this index name happens to not be unique, right?

Reply:
Right. I could have it validate that the index name is available at the time (but not in a tx, so there's still a small race condition there).

e